website_v2/gcloud-powershell-website-v2/www/static/app/app.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+    var app = angular.module('powershellSite', ['ngRoute']);
+
+    app.config(function($routeProvider, $locationProvider) {
+        $routeProvider
+            .when('/', {
+                templateUrl: 'static/templates/home.html',
+                controller: 'ReferenceCtrl',
+                controllerAs: 'ref',
+                reloadOnSearch: false
+            })
+            .when('/:product', {
+                templateUrl: 'static/templates/product.html',
+                controller: 'ReferenceCtrl',
+                controllerAs: 'ref',
+                reloadOnSearch: false
+            })
+            .when('/:product/:cmdlet', {
+                templateUrl: 'static/templates/cmdlet.html',
+                controller: 'ReferenceCtrl',
```

Comment:
I would fully name the controllers themselves with *Controller.  So ReferenceController, but it's up to you.